common/testutils/test_random.go

```diff
@@ -0,0 +1,108 @@
+package testutils
```

Comment:
How about putting this in another package, like `testutils/random` instead?

Reply:
[09e6390](https://github.com/Layr-Labs/eigenda/pull/976/commits/09e63904bb2ac75517d8e9ff4f0c25ea31cda94d)